zerver/data_import/rocketchat.py

```diff
@@ -634,9 +634,10 @@ def list_reactions(reactions: Dict[str, Dict[str, Any]]) -> List[Dict[str, Any]]
             usernames = reactions[react_code]["usernames"]
 
             for username in usernames:
-                rc_user_id = username_to_user_id_map[username]
-                user_id = user_id_mapper.get(rc_user_id)
-                reactions_list.append({"name": name, "user_id": user_id})
+                if username in username_to_user_id_map:
```

Comment:
I'd prefer to do an `if username not in username_to_user_id_map: continue` block, since that involves less indentation and will make it more natural to add a comment for that block noting that this appears possible based on production data.

Reply:
Adjusted as requested